tests/robot-tests/tests/admin_and_public/bau/prelease_visibility.robot

```diff
@@ -0,0 +1,223 @@
+    user clicks link  Edit this release
+
+Add public prerelease access list
+    [Tags]  HappyPath
+    user clicks link  Pre-release access
+    user waits until h2 is visible  Manage pre-release user access
+    user creates public prerelease access list  Initial test public access list
+
+Update public prerelease access list
+    [Tags]  HappyPath
+    user updates public prerelease access list  Updated test public access list
+
+Add meta guidance to ${PUBLICATION_NAME} subject
+    [Tags]  HappyPath
+    user clicks link  Data and files
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document  90
+    user enters text into element  id:metaGuidanceForm-content  Test meta guidance content
+    user waits until page contains accordion section  UI test subject
+    user enters text into meta guidance data file content editor  UI test subject  metaguidance content
+    user clicks button  Save guidance
+
+Go to "Sign off" page
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until h2 is visible  Sign off
+    user waits until page contains button  Edit release status
+
+Approve release and wait for it to be Scheduled
+    [Tags]  HappyPath
+    ${day}=         get current datetime  %-d  2
+    ${month}=       get current datetime  %-m  2
+    ${month_word}=  get current datetime  %B  2
+    ${year}=        get current datetime  %Y  2
+
+    user clicks button  Edit release status
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote     Approved by prerelease UI tests
+    user waits until page contains element   xpath://label[text()="On a specific date"]/../input
+    user clicks radio   On a specific date
+    user waits until page contains   Publish date
+    user enters text into element  id:releaseStatusForm-publishScheduled-day    ${day}
+    user enters text into element  id:releaseStatusForm-publishScheduled-month  ${month}
+    user enters text into element  id:releaseStatusForm-publishScheduled-year   ${year}
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-month   1
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-year    2001
+    user clicks button   Update status
+
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user checks summary list contains  Scheduled release  ${day} ${month_word} ${year}
+    user checks summary list contains  Next release expected  January 2001
+    user waits for release process status to be  Scheduled  90
+
+Navigate to prerelease page
+    [Tags]  HappyPath
+    ${current_url}=  get location
+    ${RELEASE_URL}=  remove substring from right of string  ${current_url}  /status
+    set suite variable  ${RELEASE_URL}
+    user goes to url   ${RELEASE_URL}/prerelease
+
+Validate prerelease has not started
```

Comment:
We don't need this test case (and the preceding one) in this suite, since `tests/admin/bau/prerelease.robot` already checks whether the `Pre-release access is not yet available` page displays correctly.

I think the test we need is to check the prerelease page isn't accessible before the release has been scheduled.